ilc/server/serveStatic.js

```diff
@@ -17,10 +17,15 @@ module.exports = function (isProduction) {
             publicPath: '/',
             headers: {
                 "Access-Control-Allow-Origin": "*",
+                "Content-Encoding": "gzip",
```

Comment:
it's development, and now gzipping works by default on prod

Reply:
Yeah, I missed it
I'll add it
Thanks!

Reply:
Added